charts/opentelemetry-demo/values.yaml

```diff
@@ -0,0 +1,116 @@
+observability:
+  otelcol:
+    enabled: true
+  jaeger:
+    enabled: true
+
+image:
+  repository: otel/demo
+  pullSecrets: []
+
+serviceAccount: ""
+
+components:
+  redis:
+    enabled: true
+    image: redis:alpine
+    ports:
+    - name: redis
+      value: 6379
+
+  adService:
+    enabled: true
+    servicePort: 8080
+
+  cartService:
+    enabled: true
+    servicePort: 8080
+    env:
+    - name: ASPNETCORE_URLS
+      value: http://*:8080
+
+  checkoutService:
+    enabled: true
+    servicePort: 8080
+
+  currencyService:
+    enabled: true
```

Comment:
If this component is disabled, what is the outcome for the checkout-service and frontend-service, both which have a dependency on the currency-service in `_deployment.tpl`?

Reply:
For example, the frontend service has many dependencies (including currencyService), and if currencyService is disabled, the frontend service will still run normally. But when the order is placed, it will fail to place the order. In jaeger, you can see that frontend tries to access the currency but returns 500.